Auth0/SafariWebAuth.swift

```diff
@@ -76,6 +76,20 @@ class SafariWebAuth: WebAuth {
         return self
     }
 
+    func responseType(_ responseType: [ResponseType]) -> Self {
+        if !responseType.contains(.code) {
+            _ = self.usingImplicitGrant()
+        }
+        let response = responseType.reduce(String()) { $0 + $1.rawValue + " "}
```

Comment:
It's not as easy as that due to it being an enum and requiring rawValue, perhaps with the suggestion of using `OptionSet` it will flow more easily. I'll look into it.